lang/src/main/scala/scalasvg/lang/typeclass/Equal.scala

```diff
@@ -0,0 +1,9 @@
+package scalasvg.lang.typeclass
+
+trait Equal[A] {
+
+  def equal(x: A, y: A): Boolean
+
+  final def notEqual(x: A, y: A): Boolean = !equal(x, y)
+  
+}
```

Comment:
Please, add a companion object that extends summoner `InstanceSummoner`. It enables us to summon `Equal` instances by a little bit simpler - `Equal[SomeType]`, instead of `summon[Equal[SomeType]]`.